src/lib/components/inbox/bids/index.tsx

```diff
@@ -0,0 +1,54 @@
+import * as React from "react"
+import * as Relay from "react-relay"
+import { View } from "react-native"
+import { LargeHeadline } from "../typography"
+
+class ActiveBids extends React.Component<any, any> {
+  render() {
+    console.log(this.props.me)
+    return (
+      <View>
+        <LargeHeadline>Active Bids</LargeHeadline>
+      </View>
+    )
+  }
+}
+
+
+export default Relay.createContainer(ActiveBids, {
+  fragments: {
+    me: () => Relay.QL`
+      fragment on Me {
+        lot_standings(active_positions: true) {
+          active_bid {
+            id
+            display_max_bid_amount_dollars
+            max_bid {
+              cents
+              display
+            }
+            sale_artwork {
+              lot_label
+              lot_number
+              position
+              highest_bid {
+                cents
+                display
+              }
+              artwork {
+                id
+                title
+                image {
+                  image_url
+                }
+                artist {
```

Comment:
I think we should use the [`artist_names` string](https://github.com/artsy/metaphysics/blob/e091e95d7186a6f58b559477a51d37fbb284400e/schema/artwork/index.js#L74) here instead - sometimes artworks have multiple artists or cultural makers.